src/main/java/racingCar/domain/TryCount.java

```diff
@@ -0,0 +1,25 @@
+package racingCar.domain;
+
+public class TryCount {
+
+    private int tryCount;
+
+    public TryCount(int tryCount) {
+        this.tryCount = tryCount;
+    }
+
+    protected void decrease() {
+        validateIsCountZero();
+        --this.tryCount;
```

Comment:
값 객체 도출까지 해주셨네요 👍🏻 

값 객체는 다음과 같은 형태 불변으로 사용하는 경우가 많아요. 

```suggestion
    protected void decrease() {
        validateIsCountZero();
        return new TryCount(--this.tryCount);
```

[[ 이펙티브 자바 ] 아이템 17 변경 가능성을 최소화하라](https://velog.io/@meme2367/%EC%9D%B4%ED%8E%99%ED%8B%B0%EB%B8%8C-%EC%9E%90%EB%B0%94-%EC%95%84%EC%9D%B4%ED%85%9C-17-%EB%B3%80%EA%B2%BD-%EA%B0%80%EB%8A%A5%EC%84%B1%EC%9D%84-%EC%B5%9C%EC%86%8C%ED%99%94%ED%95%98%EB%9D%BC)를 참고해시고 너무 어렵게 느껴지신다면 가볍게 지나가셔도 좋습니다!

Reply:
오 이런 형태가 있다는 건 아예 처음 알았네요.
좋은 지식 감사합니다.